htcondor/htcondor_query.py

```diff
@@ -335,6 +331,7 @@ def eval_classad_expr(classads, format_list=None):
                     # attribute exists in the fetched classad
                     dict_el[attr] = classad[attr]
             except Exception:
+                logger.exception("Cannot find attribute for htcondor_query.eval_classad_expr")
```

Comment:
since this looks like a defensive `except`, can you add `# pragma: no cover` to the `except Exception:` line?